.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 dist
+package-lock.json
+.idea
```

Comment:
I've personally received a nope on this (for vim/vscode though) form @krzkaczor once before 😅 but his arguments are right: you can add those to your [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) -- otherwise soon we'll end up with dozens of gitignore entries for every OS & editor combination out there.